Woof/WoofTests/Model/LocalStorage/LocalStorageSaveTests.swift

```diff
@@ -0,0 +1,33 @@
+import XCTest
+
+final class LocalStorageSaveTests: XCTestCase {
+    var localStorage: LocalStorage!
+
+    override func setUp() {
+        let storage = LocalStorage()
+        localStorage = storage
+
+        UserDefaults.standard.removePersistentDomain(forName: Bundle.main.bundleIdentifier!)
+    }
+
+    override func tearDown() {
+        localStorage = nil
+        super.tearDown()
+    }
+
+    func testSaveAndLoadValueAreEqual() {
```

Comment:
It's not clear what values are equal
```suggestion

    func testSaveAndLoadValueAreEqualForTheProvidedKey() {
```

Reply:
Ok